update-github-pages.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
 cd "$HOME"
+
+git config credential.helper "store --file=.git/credentials"
+echo "https://${GITHUB_TOKEN}:@github.com" > .git/credentials
```

Comment:
That is a good question. It is stolen from https://github.com/t0yv0/t0yv0.github.io/blob/master/deploy.sh#L4 but otherwise stays unused as I switched to the silent git push approach.